public/429.html

```diff
@@ -3,15 +3,15 @@
 <head>
   <title>Rate Limited | Login.gov</title>
   <meta name="viewport" content="width=device-width,initial-scale=1">
-  <link rel="stylesheet" href="../css/static.css">
+  <link rel="stylesheet" href="css/static.css">
 </head>
 <body>
   <div class="site-wrapper">
     <div class="site-wrapper-inner">
       <div class="cover-container">
         <div class="masthead clearfix">
           <div class="inner">
-            <img class="masthead-brand" src="../images/logo-white.svg" width="150">
+            <img class="masthead-brand" src="images/logo-white.svg" width="150">
```

Comment:
With or without the leading slash should work since its relative to base url, the other error pages use`css/static.css` so I made this one the same for consistency, otherwise we should change all of them to have the leading slash.